src/list/FilterSidebar.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+
+import styled from 'styled-components';
+import { Label, SearchInput, CheckboxGroup, IndeterminateCheckbox, List, ListRow } from '@sailthru/stui-components';
+import { fontSizeSmall, fontSizeMedium, fontSizeLarge, fontWeightBold, grayTint90 } from '@sailthru/design-tokens';
+
+const FilterContainer = styled.div`
+    margin-top: 1.875em;
+    padding: 1em;
+    height: 100vh;
+    background-color: ${grayTint90};
+    font-size: ${fontSizeMedium};
+`;
+
+const OptionGroupContainer = styled.div`
+    margin-top: 1em;
+    font-size: ${fontSizeMedium};
+`;
+
+const ListRowContainer = styled.div`  
+    cursor: pointer;
+    li {
+        padding-left: 0rem;
+    }
+`;
+
+export default function FilterSidebar({ onChange }) {
+    const [ searchValue, setSearchValue ] = useState('');
+    const [ selectedMode, setSelectedMode ] = useState([]);
+    const [ selectedType, setSelectedType ] = useState([]);
+    const [ typesList, setTypesList ] = useState([]);
+
+    const modes = ['HTML'];
+    const types = ['Signup', 'Manage', 'Other'];
+    
+    function onClickMode(event) {
+        const selectedMode = event.target.attributes.value.value;
+        setSelectedMode(selectedMode);
+        onChange({types: typesList, mode: selectedMode, name: searchValue});
+    }
+
+    function onChangeType({newValue}) {
+        setSelectedType(newValue);
+        const typeList = [];
+        newValue.forEach(value => {
+            typeList.push(types[value]);
+        })
+        setTypesList(typeList);
+        onChange({types: typeList, mode: selectedMode, name: searchValue});
+    }
+
+    function onChangeSearchValue(newValue) {
+        setSearchValue(newValue);
+        onChange({types: typesList, mode: selectedMode, name: newValue});
+    }
+
+    return (
+        <FilterContainer>
+            <OptionGroupContainer>
+                <Label fontsize={fontSizeLarge} style={{fontWeight: fontWeightBold}}>
+                    FILTER
+                </Label>
+                <SearchInput
+                    value={searchValue}
+                    placeholder="Search"
+                    onChange={onChangeSearchValue}
+                />
+            </OptionGroupContainer>
+            <OptionGroupContainer>
+                <Label fontsize={{ fontSizeSmall }} style={{ fontWeight: fontWeightBold }}>
+                    Type
+                </Label>
+                <List selected={selectedMode} >
+                {modes.map((value, key) => (
+                    <ListRowContainer value={value} key={key}>
+                        <ListRow onClick={onClickMode}>
+                            <i className="fal fa-code" title="HTML Page" />
+                            &nbsp;{value}  
+                        </ListRow>
+                    </ListRowContainer>
```

Comment:
blocking: `{value} key={key}` within `ListRowContainer` causes an error and does not pass the mode. Move `{value} key={key}` to `ListRow`
```suggestion
<ListRowContainer >
                        <ListRow value={value} key={key} onClick={onClickMode}>
                            <i className="fal fa-code" title="HTML Page" />
                            &nbsp;{value}
                        </ListRow>
                    </ListRowContainer>
```